modules/oadp-ceph-cephfs.adoc

```diff
@@ -0,0 +1,180 @@
++
+[source,terminal]
+----
+$ oc get storageClass -A  -o jsonpath='{range .items[*]}{"Name: "}{.metadata.name}{"  "}{"annotations: "}{.metadata.annotations}{"\n"}{end}'
+----
+
+. Create a Data Protection Application (DPA) CR similar to the following.
++
+.Example DPA CR
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <my-bucket>
+          prefix: velero
+       provider: aws
+    configuration:
+      restic:
+        enable: false  <1>
+      velero:
+        defaultPlugins:
+          - openshift
+          - aws
+          - csi
+          - vsm
+    features:
+      dataMover:
+        credentialName: <restic-secret-name> <2>
+        enable: true <3>
+        volumeOptionsForStorageClasses:
+          ocs-storagecluster-cephfs:
+            sourceVolumeOptions:
+              accessMode: ReadOnlyMany
+              cacheAccessMode: ReadWriteMany
+              cacheStorageClassName: ocs-storagecluster-cephfs
+              storageClassName: ocs-storagecluster-cephfs-shallow
+----
+<1> There is no default value for the `enable` field. Valid values are `true` or `false`.
+<2> Use the Restic `Secret` that you created when you prepared your environment for working with OADP 1.2 Data Mover and Ceph. If you do not, then the CR will use the default value `dm-credential` for this parameter.
+<3> There is no default value for the `enable` field. Valid values are `true` or `false`.
+
+. To back up data, complete the following steps:
+
+.. Create a `Backup` CR similar to the following:
++
+.Example `Backup` CR
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: <backup-name>
+  namespace: <protected-ns>
+spec:
+  includedNamespaces:
+  - <app-ns>
+  storageLocation: velero-sample-1
+----
+
+.. Monitor the progress of the `VolumeSnapshotBackup` CRs by doing the following:
+... To check the progress of all the `VolumeSnapshotBackup` CRs, run the following command:
++
+[source, terminal]
+----
+$ oc get vsb -n <app-ns>
+----
+
+... To check the progress of a specific `VolumeSnapshotBackup` CR, run the following command:
++
+[source,terminal]
+----
+$ oc get vsb <vsb-name> -n <app-ns> -ojsonpath="{.status.phase}`
+----
+
+.. Wait several minutes until the `VolumeSnapshotBackup` CR has the status `Completed`.
+.. Verify that there is at least one snapshot in the object store that is given in the Restic `secret`. You can check for this snapshot in your targeted `BackupStorageLocation` storage provider that has a prefix of `/<OADP-namespace>`.
+
+. To restore data, complete the following steps:
+
```

Comment:
@akarol @tareqalayan ASAP I need someone in QE to verify that lines 129-143 below is a correct way to
"Make sure the application namespace is deleted, as well as any volumeSnapshotContents that were created during backup."  David Vaanunu is the guy who knows this best, but I can't tag him here (will try in Jira)